### PR TITLE
Test publish test result

### PR DESCRIPTION
**Pull request by Sergey-SQA**

### CODE REVIEW

File: .github/workflows/schedule-test-run-pipeline.yml [hunk 1]
```diff
@@ -20,4 +19,10 @@ jobs:
     - name: Build
       run: dotnet build
     - name: Test
-      run: dotnet test
+      run: dotnet test --logger "trx;LogFileName=TestResults.trx"
+    - name: Parse Trx files
```
Comment (tigran201198) [inline .github/workflows/schedule-test-run-pipeline.yml:23]: You can add upload-artifact task that will publish html test report we created together.

File: NunitTrelloTest/Tests/BaseTest.cs [hunk 2]
```diff
@@ -1,23 +1,66 @@
...
+            }
+            testClassHolder = reportHolder.CreateTest(TestContext.CurrentContext.Test.ClassName);//create test class - left table
+            client = new("https://api.trello.com");
+        }
+
+        [SetUp]
+        public void SetUpTestMetadataHolderBeforeEachTest()
+        {
+            var testMethodName = TestContext.CurrentContext.Test.Name;
+            var currentTestMethod = GetType().GetMethods().First(method => testMethodName.StartsWith(method.Name));
```
Comment (tigran201198) [inline NunitTrelloTest/Tests/BaseTest.cs:43]: You can create a ReflectionUtils class and create extension methods for searching for types and attributes:
1. .GetMethods().First(method => testMethodName.StartsWith(method.Name));
2. (AttributeName) Attribute.GetCustomAttribute(currentTestMethod, typeof(AttributeName))

File: NunitTrelloTest/Tests/Boards/GetABoardTests.cs [hunk 3]
```diff
@@ -9,30 +9,43 @@ namespace NunitTrelloTest.Tests.Boards
     public class GetABoardTests: BaseTest
     {
         [Test]
+        [Author("Sergey Polushin")]
+        //[Igonre("Bug: JIRA_ID-123")]
```
Comment (tigran201198) [inline NunitTrelloTest/Tests/Boards/GetABoardTests.cs:13]: If you comment some code, don't include it to PR.

File: NunitTrelloTest/Tests/Members/GetBoardsThatMemberBelongsToTests.cs [hunk 4]
```diff
@@ -7,22 +8,36 @@ namespace NunitTrelloTest.Tests.Members
     public class GetBoardsThatMemberBelongsToTests: BaseTest
     {
         [Test]
+        [Author("Sergey Polushin")]
```
Comment (tigran201198) [inline NunitTrelloTest/Tests/Members/GetBoardsThatMemberBelongsToTests.cs:11]: You can add your name to a protected constant in BaseTest class. So, if test author name changes, you can update it in one single place.

**Comment by Sergey-SQA** [comment]
Merged

**Pull request merged**